app/assets/javascripts/discourse/models/bookmark.js.es6

```diff
@@ -0,0 +1,24 @@
+import discourseComputed from "discourse-common/utils/decorators";
+import { none } from "@ember/object/computed";
+import { ajax } from "discourse/lib/ajax";
+import { Promise } from "rsvp";
+import RestModel from "discourse/models/rest";
+
+const Bookmark = RestModel.extend({
+  newBookmark: none("id"),
+
+  @discourseComputed
```

Comment:
I would prefer computed here, discourseComputed should only be used when you have params, as its main advantage is to provide computed properties as params.